src/main/java/org/wiremock/integrations/testcontainers/WireMockContainer.java

```diff
@@ -47,9 +47,14 @@
 public class WireMockContainer extends GenericContainer<WireMockContainer> {
 
     public static final String OFFICIAL_IMAGE_NAME = "wiremock/wiremock";
-    private static final String WIREMOCK_2_LATEST_TAG = "2.35.0";
+    private static final String WIREMOCK_2_LATEST_TAG = "2.35.0-1";
+    private static final String WIREMOCK_3_LATEST_TAG = "3.0.1-1";
     /*package*/ static final String WIREMOCK_2_MINIMUM_SUPPORTED_VERSION = "2.0.0";
 
+    /**
+     * @deprecated Not really guaranteed to be latest. Will be reworked
+     */
+    @Deprecated
```

Comment:
I would remove it, since we haven't yet released - we could afford removing old code

Reply:
Agreed